[Blog] ServiceMeshCon 2022 talk with Intel - MeshMark (#2792)
layer5io/layer5: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx

---

> title: "MeshMark: Service Mesh value measurement"

Reviewer: ```suggestion
title: "MeshMark: Cloud Native Value Measurement"
```

date: 2022-05-17 10:30:05 -0530
author: Gaurav Chadha
thumbnail: ./banner.png

> description: "ServiceMeshCon 2022 talk with Intel"

Reviewer: ```suggestion
description: "An open index for measuring performance of cloud native infrastructure in context of the value provided to your business."
```

type: Blog

> category: "MeshMark"

Reviewer: ```suggestion
category: "Service Mesh Performance"
```

tags:
 - MeshMark
 - ServiceMeshCon
published: true
---

import { BlogWrapper } from "../../Blog.style.js";
import Blockquote from "../../../../reusecore/Blockquote";

> import slidesTAG from "../../../events/kubecon-china-2021/cncf-tag-network-and-service-mesh-working-group-kubecon-china-2021-lee-calcote-ken-owens.pdf";

Reviewer: Thanks

    <img src={MUE} alt="MeshMark MUE" style="width:50%;float:left;padding-right:1rem;" /> And you would want to see if the CPU performance as a ratio to the available resources is a loss or a gain. So CPU performance, raw loss over total CPU is our MUE one.
    And that's just one minus CPU utilization over 100. That's a very simple ratio and if you see on the slide, the graph shows you that as the latency increases, your Mue lowers. And so that's a very good indicator that your efficiency of your infrastructure is not very good because your latencies are increasing as your QPS increases.
    So like this you can measure and create other MUEs. We will look at how you can visualize this within an environment and so let's look at the demonstration. So let's jump into a sibling CNCF project called Meshery.
    Meshery is a cloud native management plane. Users of Meshery can configure their Kubernetes deployments any and every service mesh as well as on board and off board their workloads onto any given mesh.
</p>    

<img src={MeshMapDemo} alt="MeshMap demo" style="width:40%;float:right;"  />

> <h3>Lee demonstrates MeshMap with an example of console application</h3>

Reviewer: ```suggestion
<h3>Lee demonstrates MeshMap with an example Consul application</h3>
```

<p>

> Let's take an example workload a console application, load it into the visual designer, take a look at the service splitting functionality of console and note in this case we're assigning a weight of three when we can change that to four to derive its mesh Mark which is a mesh utilization efficiency calculation of the efficiency by which that network function is being performed.

Reviewer: ```suggestion
Let's take an example workload a  Consul application, load it into the visual designer, take a look at the service splitting functionality of console and note in this case we're assigning a weight of three when we can change that to four to derive its mesh Mark which is a mesh utilization efficiency calculation of the efficiency by which that network function is being performed.
```
